__tests__/store/api.spec.js

```diff
@@ -0,0 +1,93 @@
+import { NodeService } from '../../src/infrastructure';
+import api from '../../src/store/api';
+import { createLocalVue } from '@vue/test-utils';
+import { stub } from 'sinon';
+import Vuex from 'vuex';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
+describe('store/api', () => {
+	describe('action loadNodeList should', () => {
+		// Mock
+		const mockApiNodeList = [{
+            publicKey: 'B44144608FCC7CB60B55505849A3E07A8837549DDC886968B30B0568976C0B41',
+            roles: 'Peer Api node',
+            friendlyName: 'mock1',
+            networkGenerationHashSeed: '57F7DA205008026C776CB6AED843393F04CD458E0AA2D9F1D5F31A402072B2D6',
+            port: 7900,
+            networkIdentifier: 152,
+            host: 'mock1.com',
+            nodePublicKey: 'B44144608FCC7CB60B55505849A3E07A8837549DDC886968B30B0568976C0B41',
+            address: 'TCTSR5BB4WSRZ2TPFUVSG2HR6BGYJ7JUK2ZDGLA',
+            rolesRaw: 3,
+            network: 'TESTNET',
+            apiEndpoint: 'https://mock1.com:3001'
+        },
+        {
+            publicKey: '9463FD29F98D47EBE8CE697F29AA060A45912845225352AFBD31961F742AFCC6',
+            roles: 'Peer Api node',
+            friendlyName: 'mock2',
+            networkGenerationHashSeed: '57F7DA205008026C776CB6AED843393F04CD458E0AA2D9F1D5F31A402072B2D6',
+            port: 7900,
+            networkIdentifier: 152,
+            host: 'mock2.com',
+            nodePublicKey: '9463FD29F98D47EBE8CE697F29AA060A45912845225352AFBD31961F742AFCC6',
+            address: 'TAINMOVAIHUWFN6NFLBR7EGTVJH6W3MBE4FOVDI',
+            rolesRaw: 3,
+            network: 'TESTNET',
+            apiEndpoint: 'https://mock2:3001'
```

Comment:
since there is a lot of common properties, you can have a shared object and then set with spread ... operator